### DISCUSSION TITLE
Where is the DOCUMENTATION and EXAMPLEs of each and every EXTENSION ?

### DISCUSSION THREAD

**Discussion by alanscot** [TurboWarp/extensions#509] [Q&A]
Eh?

**Comment by David-Orangemoon** [comment]
What

**Comment by ghost** [comment]
Maybe someday it will be... By the way, you can speed up this moment by making several documentations with examples yourself.

**Comment by DNin01** [comment]
I'm pretty good at explaining what I know in documentation form, so I'd be interested in writing documentation for any extensions people have questions about. I just wrote a page for Bitwise, for instance. Learning about that was pretty fun!

**Reply by alanscot** [reply]
Also can your write the extension EXAMPLE  ( probably CLOUDLINK extension) to use "fetch from a URL"( block) maybe ( Do an http GET from/to a .php file from/to a web server) ?

**Comment by alanscot** [comment]
Thanks for offer.Could you do some big examples for the 2D PHYSICS extension, please ?Alan
    On Sunday, June 4, 2023 at 11:49:53 AM PDT, DNin01 ***@***.***> wrote:  
 
 


I'm pretty good at explaining what I know in documentation form, so I'd be interested in writing documentation for any extensions people have questions about. I just wrote a page for Bitwise, for instance. Learning about that was pretty fun!

**Reply by DNin01** [reply]
I made it! #532

**Comment by alanscot** [comment]
I'm sorry I clicked on every tab and every link and couldn't find the documentation :( 
    On Sunday, June 4, 2023 at 05:43:14 PM PDT, DNin01 ***@***.***> wrote:  
 
 


I made it! #532

**Reply by DNin01** [reply]
It's still in review so it's not available on the website yet. You can open the pull request to check it out.

**Reply by DNin01** [reply]
It's live: https://extensions.turbowarp.org/box2d.html
